pages/admin/PersonDetailsView.page.ts

```diff
@@ -6,7 +6,6 @@ export class PersonDetailsViewPage{
     readonly button_pwChange: Locator;
     readonly button_pwReset: Locator;
     readonly text_pwResetInfo: Locator; 
-    readonly icon_pwVisible: Locator;
```

Comment:
War das beabsichtigt? Das hat glaube ich nichts mit der Profile-Seite zu tun. Diesser locator wird in dem Testfile workflow.spec.ts verwendet.

Reply:
Ach nee, hab mich versehen, der locator wird nicht verwendet und kann raus.